app/i18n/ja/admin.php

```diff
@@ -187,33 +187,33 @@
 			),
 			'title' => 'ユーザー登録',
 		),
-		'sensitive-parameter' => 'Sensitive parameter. Edit manually in <kbd>./data/config.php</kbd>',	// TODO
+		'sensitive-parameter' => 'センシティブなパラメーターです。<kbd>./data/config.php</kbd> を手動で編集',
 		'tos' => array(
-			'disabled' => 'is not given',	// TODO
-			'enabled' => '<a href="./?a=tos">is enabled</a>',	// TODO
-			'help' => 'How to <a href="https://freshrss.github.io/FreshRSS/en/admins/12_User_management.html#enable-terms-of-service-tos" target="_blank">enable the Terms of Service</a>',	// TODO
+			'disabled' => 'disabled',	// IGNORE
+			'enabled' => '<a href="./?a=tos">enabled</a>',	// IGNORE
+			'help' => '<a href="https://freshrss.github.io/FreshRSS/en/admins/12_User_management.html#enable-terms-of-service-tos" target="_blank">利用規約を有効にする方法</a>',
 		),
 		'websub' => array(
-			'help' => 'About <a href="https://freshrss.github.io/FreshRSS/en/users/WebSub.html" target="_blank">WebSub</a>',	// TODO
+			'help' => '<a href="https://freshrss.github.io/FreshRSS/en/users/WebSub.html" target="_blank">WebSubについて</a>',
 		),
 	),
 	'update' => array(
 		'_' => 'システムアップデート',
 		'apply' => '適用',
-		'changelog' => 'Changelog',	// TODO
+		'changelog' => '変更履歴',
 		'check' => 'アップデートを確認する',
-		'copiedFromURL' => 'update.php copied from %s to ./data',	// TODO
+		'copiedFromURL' => 'update.php が %s から ./data にコピーされました。',
 		'current_version' => '現在のバージョンは',
 		'last' => '最近の検証',
-		'loading' => 'Updating…',	// TODO
+		'loading' => '更新中...',
 		'none' => '更新を適用できません',
 		'releaseChannel' => array(
-			'_' => 'Release channel',	// TODO
-			'edge' => 'Rolling release (“edge”)',	// TODO
-			'latest' => 'Stable release (“latest”)',	// TODO
+			'_' => 'Release channel',	// IGNORE
+			'edge' => 'Rolling release (“edge”)',	// IGNORE
+			'latest' => 'Stable release (“latest”)',	// IGNORE
```

Comment:
No Japanese words here?

Reply:
@math-GH 
The alphabet is also understood, but katakana certainly seemed more useful, so I will revise it.